getstarted.md

```diff
@@ -22,6 +22,70 @@ with Hadoop, as well as extensions to that foundation that make for totally
 awesome demos. You'll also find a few videos related to our work and of course,
 how to contact us!
 
+First, though, let's just take a look at how straightforward it is to get started with very complex software. You can do it during a lunch break:
```

Comment:
Do not think this is the right place to put this tutorial. I propose we move this to a tutorials section so that we get into the habit of adding more examples and link from our landing page.